### ISSUE TITLE
getReport accessor

### ISSUE THREAD

**Issue by sumkincpp** [emcvipr/mnrcli#3]
Possible enhancement: also would be great to be able to generate Reports via `getReport` Report Generation Interface

**Comment by bauagonzo** [comment]
It is a different wsdl. But we could develop another utility.
What would be your use-case ?

**Comment by sumkincpp** [comment]
Usual case of retrieving data from existent report where not only metrics, but some additional formulas/expansions are used.

**Comment by bauagonzo** [comment]
The problem is with getReport you have to pass the entire report definition (and sometimes change a couple of tags). 
You have an example of how to use getReport in : https://github.com/eastsidegeek/INQPlus

**Comment by sumkincpp** [comment]
I agree that this API works with the report definition, so it isn't capable for solving this task.

I hadn't played a lot with ViPR SRM web-services, but are there any other API-based ways for getting reports data? (Real-time, Scheduled Reports, Stored Reports)
